visual-regressions/README.md

```diff
@@ -0,0 +1,23 @@
+## What is this?
```

Comment:
Could you please move this into the https://github.com/securityscorecard/design-system/blob/main/TESTING.md#visual-regression

Reply:
or just reference it there